style.css

```diff
@@ -282,10 +273,134 @@ header {
   display: flex;
   justify-content: center;
   margin-left: 10%;
-  padding: 24px 80px 24px 80px;
+  padding: 24px 80px;
   width: 120px;
 }
 
+.contact-text {
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  color: #172b4d;
+  padding: 40px 20px;
+}
+
+.get-btn {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed),  as shown in the Figma template below:

![image](https://user-images.githubusercontent.com/52934128/219883731-3d32d6f5-53ac-4128-a0c8-1a7595de03e5.png)

the button on the pressed state has the `background: #008552;` and on hover has the `background: #36B37F;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #008552;
}
```
**for hover state**
```
button:hover {
    background-color: #36B37F;
}
```